webapp/channels/src/components/suggestion/suggestion_box/suggestion_box.jsx

```diff
@@ -396,7 +396,10 @@ export default class SuggestionBox extends React.PureComponent {
             return;
         }
 
-        const suffix = text.substring(caret);
+        let suffix = text.substring(caret);
+        if (term.endsWith('join')) {
+            suffix = '~';
```

Comment:
We don't want to use the `suffix` for this because that contains all the pre-existing text after the keyboard caret. For example, if the user goes back to add an emoji earlier in this text
```
this : is some text
     ^ the caret is after this colon
```
then the suffix will be ` is some text`